schedule.go

```diff
@@ -295,7 +309,6 @@ func (c *Client) DeleteOverrideWithContext(ctx context.Context, scheduleID, over
 
 // ListOnCallUsersOptions is the data structure used when calling the ListOnCallUsers API endpoint.
 type ListOnCallUsersOptions struct {
-	APIListObject
```

Comment:
For all of these `APIListObject` deletes in this file, it looks like they were added to these types by mistake. They are not part of the API specification.